man/systemd-resolve.xml

```diff
@@ -321,6 +321,46 @@
         <listitem><para>Shows the global and per-link DNS settings in currently in effect.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--set-dns=SERVER</option></term>
+        <term><option>--set-domain=DOMAIN</option></term>
+        <term><option>--set-llmnr=MODE</option></term>
+        <term><option>--set-mdns=MODE</option></term>
+        <term><option>--set-dnssec=MODE</option></term>
+        <term><option>--set-nta=DOMAIN</option></term>
+
+        <listitem><para>Set per-interface DNS configuration. These switches may be used to configure various DNS
+        settings for network interfaces that aren't managed by
```

Comment:
Should we stress that the specified interface is unmanaged? I was very confused by getting the following result.
```
$ sudo systemd-resolve -i enp0s25 --set-dns=8.8.8.8
Failed to set DNS configuration: Link enp0s25 is managed.
```